packages/back-end/src/events/notifiers/FeatureUpdatedNotifier.ts

```diff
@@ -0,0 +1,87 @@
+import { EventEmitter } from "node:events";
+import Agenda, { Job, JobAttributesData } from "agenda";
+import { getAgendaInstance } from "../../services/queueing";
+import { ApiFeatureInterface } from "../../../types/api";
+import { getApiFeatureObjForFeatureIdOrganizationId } from "../../services/features";
+import { FeatureUpdatedNotificationEvent } from "../base-events";
+import {
+  APP_NOTIFICATION_EVENT_EMITTER_NAME,
+  NotificationEventHandler,
+} from "../base-types";
+import { getEventEmitterInstance } from "../../services/event-emitter";
+
+interface Notifier {
+  enqueue(featureId: string, organizationId: string): Promise<void>;
+}
+
+interface EnqueuedData extends JobAttributesData {
+  featureId: string;
+  organizationId: string;
+}
+
+/**
+ * Handle the notifications sent by the FeatureUpdatedNotifier.
+ * Example usage:
+ *
+ *  const myHandler: FeatureUpdatedNotificationHandler = async (payload) => {
+ *    const feature: ApiFeatureInterface = payload.data;
+ *    // Do async things
+ *  };
+ */
+export type FeatureUpdatedNotificationHandler = NotificationEventHandler<
+  FeatureUpdatedNotificationEvent,
+  void
+>;
+
+export class FeatureUpdatedNotifier implements Notifier {
```

Comment:
This class is implemented when we update an event. This is the pattern I will be following for all other event types going forward so please review this and let me know if I missed anything.